src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
@8BitPimp just a side note about what you said: take a look at https://github.com/OpenRCT2/OpenRCT2 then.

Main dev's screencast: https://www.youtube.com/watch?v=0AMIE2sj2AA